Adapt table sso (#475)
Software-Improvement-Group/sigridci: docs/organization-integration/usermanagement.md

- Audience or Identifier (Entity) ID: urn:amazon:cognito:sp:eu-central-1_hwh9zdyCY
- Reply URL: https://auth.sigrid-says.com/saml2/idpresponse

With the following 3 Attributes & Claims:

> | Your user | Sigrid expects the long url with namespace and SAML attribute name |

Reviewer: ```suggestion
| Your user | Sigrid expects the long url as SAML attribute name |
```

| ----------- | ----------|
| user email  | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress |
| user last name   | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name |
| user first name   | http://schemas.xmlsoap.org/ws/2005/05/identity/claims/given_name |

> | if your IdP requires to set an unique user identifier | please choose emailaddress |

Reviewer: ```suggestion
_Note: if your IdP requires to set an unique user identifier, please choose emailaddress_
```
